Sources/SPTPersistentCacheOptions.m

```diff
@@ -31,68 +32,89 @@
 const NSUInteger SPTPersistentCacheMinimumExpirationLimit = 60;
 
 
+#pragma mark Helper Functions
 
-#pragma mark SPTPersistentCacheOptions
+static NSUInteger SPTGuardedPropertyValue(NSUInteger proposedValue, NSUInteger minimumValue, SEL propertySelector, SPTPersistentCacheDebugCallback debugCallback);
 
-@interface SPTPersistentCacheOptions ()
-@property (nonatomic) NSString *identifierForQueue;
-@end
 
+#pragma mark - SPTPersistentCacheOptions Implementation
 
 @implementation SPTPersistentCacheOptions
 
-#pragma mark - Initializers
+#pragma mark Object Life Cycle
 
 - (instancetype)init
 {
-    return [self initWithCachePath:[NSTemporaryDirectory() stringByAppendingPathComponent:@"/com.spotify.temppersistent.image.cache"]
-                        identifier:@"persistent.cache"
-         defaultExpirationInterval:SPTPersistentCacheDefaultExpirationTimeSec
-          garbageCollectorInterval:SPTPersistentCacheDefaultGCIntervalSec
-                             debug:^(NSString *debugString) {
-                                 NSLog(@"%@", debugString);
-                             }];
-}
-
-- (instancetype)initWithCachePath:(NSString *)cachePath
-                       identifier:(NSString *)cacheIdentifier
-        defaultExpirationInterval:(NSUInteger)defaultExpirationInterval
-         garbageCollectorInterval:(NSUInteger)garbageCollectorInterval
-                            debug:(SPTPersistentCacheDebugCallback)debugCallback
-{
     self = [super init];
+
     if (self) {
+        _cachePath = [NSTemporaryDirectory() stringByAppendingPathComponent:@"/com.spotify.temppersistent.image.cache"];
```

Comment:
This default is a bit weird. However I didn’t want to change it since it’s in 1.0.0, and changing it might brake someone’s app.